site/_includes/partials/chrome-100-card-section.njk

```diff
@@ -0,0 +1,2584 @@
+              {% Img src="image/x1Los57vDga6OEMNi1dIJwZ0qvp2/2SWyoN1Cflu2i9ds1fan.svg", alt="Celebration tag",
+                  class="tags gap-top-100", width="119", height="15" %}
+              <div class="display-flex likes">
+                <a href="#" class="heart-logo" data-id="20082" data-year="2008" data-label="2008 card 2"
+                    data-category="event card" data-action="like">
+                  <svg width="20" height="19" viewBox="0 0 20 19" fill="none" xmlns="http://www.w3.org/2000/svg">
+                    <path d="M8.88659 16.6603L8.88587 16.6596C6.30081 14.3155 4.19567 12.4057 2.73078 10.6147C1.27162 8.83074 0.5 7.22576 0.5 5.5C0.5 2.69614 2.69614 0.5 5.5 0.5C7.08861 0.5 8.62112 1.24197 9.61932 2.41417L10 2.8612L10.3807 2.41417C11.3789 1.24197 12.9114 0.5 14.5 0.5C17.3039 0.5 19.5 2.69614 19.5 5.5C19.5 7.22577 18.7284 8.83077 17.2691 10.6161C15.8065 12.4055 13.7058 14.3144 11.1265 16.6584L11.1148 16.669L11.1137 16.67L10.0013 17.675L8.88659 16.6603Z" stroke="#3C4043" class="fill-like">
+                  </svg>
+                </a>
+                <span class="like-count"></span>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <div class="second-column">
+        <div class="card gap-bottom-300">
+          <div class="card-info card-top">
+            <div class="arrow-wrapper">
+              <a href="https://www.sitepoint.com/microsoft-plans-to-finally-launch-web-office-suite/" target="_blank" rel="noopener noreferrer">
+                <h1 class="card-heading">MS Office Suite for web</h1>
+                {% Img src="image/x1Los57vDga6OEMNi1dIJwZ0qvp2/dh8CuKvNIfkb8Bz00Th4.svg", alt="Arrow", class="right-arrow", width="16", height="12" %}
+              </a>
+            </div>
+            <p class="card-description">The launch brings productivity on the web to Windows users.</p>
+            <div class="display-flex gap-top-200 logos">
+              {% Img src="image/x1Los57vDga6OEMNi1dIJwZ0qvp2/cZgDIImiPwnUX3zbdBzT.svg",
+                  alt="Inspiration tag", class="tags gap-top-100", width="117", height="16" %}
+              <div class="display-flex likes">
+                <a href="#" class="heart-logo" data-id="20083" data-year="2008" data-label="2008 card 3"
+                    data-category="event card" data-action="like">
+                  <svg width="20" height="19" viewBox="0 0 20 19" fill="none" xmlns="http://www.w3.org/2000/svg">
+                  <path d="M8.88659 16.6603L8.88587 16.6596C6.30081 14.3155 4.19567 12.4057 2.73078 10.6147C1.27162 8.83074 0.5 7.22576 0.5 5.5C0.5 2.69614 2.69614 0.5 5.5 0.5C7.08861 0.5 8.62112 1.24197 9.61932 2.41417L10 2.8612L10.3807 2.41417C11.3789 1.24197 12.9114 0.5 14.5 0.5C17.3039 0.5 19.5 2.69614 19.5 5.5C19.5 7.22577 18.7284 8.83077 17.2691 10.6161C15.8065 12.4055 13.7058 14.3144 11.1265 16.6584L11.1148 16.669L11.1137 16.67L10.0013 17.675L8.88659 16.6603Z" stroke="#3C4043" class="fill-like">
```

Comment:
this icon is repeated throughout the template. Please move it to icons folder: https://github.com/GoogleChrome/developer.chrome.com/tree/main/site/_includes/icons and use `{% include path-to-icon %}` instead of repeating code.

Reply:
Done!